src/tap/grpc/server.rs

```diff
@@ -125,10 +142,24 @@ where
             }
         };
 
+        let extract = match req.extract.and_then(|ex| ExtractKind::try_from(ex).ok()) {
+            Some(ex) => ex,
+            None => {
```

Comment:
nit, no match necessary if you `.unwrap_or_else` but, as noted, this coud just be `unwrap_or_default` if we did defaulting on ExtractKind.